_pages/how-we-work/tools/github.md

```diff
@@ -93,13 +93,14 @@ Here&rsquo;s our current process to address both operational and security concer
 
 1. If the user is a member of the federal government, confirm we have an active [inter-agency agreement (IAA)](https://pages.18f.gov/iaa-forms/primer.html) or other legal document authorizing the work.
 1. If the user is a contractor, confirm we have an active and valid contract with them, or their company.
-1. Ask the collaborator(s) to go through [the setup steps](#setup).
+1. Ask the collaborator(s) to go through [the setup first three setup steps](#setup).
     * They will need to confirm they've done this before you continue.
     * They will also need to add an e-mail address to the GitHub profile so we can contact them later when doing clean-up in our org.
+    * Do not ask the admins to add the collaborator to the 18F or OPP teams as detailed in step 4.
 2. (Ask #admins-github to) [create a team](https://help.github.com/articles/setting-up-teams/) whose access we can turn off/on with one button. Separate a staff-only team from a contractor/mixed/collaborator team for a project, and name it something like `Project name - Collaborators | Skillset`. You only need to set a `parent team` for your new team if you need your team to inherit existing permissions from an existing team (for example, if this team should automatically have access to a base set of repos). If your new team is for external collaborators, you will generally not want to add a parent team.
 3. In the &ldquo;Description&rdquo; of the team, put something reasonable plus a point-of-contact email address for the collaborators.
     * Ideally this is the address of someone senior — someone you can email if issues come up and who can rally the troops.
-5. (Ask #admins-github to) [add the members](https://help.github.com/articles/maintaining-teams/).
+5. (Ask #admins-github to) [add the members to the team](https://help.github.com/articles/maintaining-teams/).
```

Comment:
Do you want to add the "https://github.com/orgs/18F/people" link that you mentioned?

Reply:
I don't think so; its simpler to do the team + org invite at the same time. Open to pushback.

Reply:
@wslack This good to merge?

Reply:
@keithrwilson For whatever my opinion is worth, I think this is OK to merge.

Reply:
my bad, I thik this is fine to merge